app/src/main/java/org/fossasia/openevent/app/data/encryption/EncryptionServiceImpl.java

```diff
@@ -102,6 +104,10 @@ private String encryptString(String toEncrypt) {
 
     @RequiresApi(api = Build.VERSION_CODES.JELLY_BEAN_MR2)
     private String decryptString(String encrypted) {
+        if (Utils.isEmpty(encrypted)) {
+            return null;
+        }
```

Comment:
Is encrypt handling the null values?

Reply:
Yes, the function encryptString() has a null check.